### ISSUE TITLE
How to handle macOS SDK requirement from Conda build?

### ISSUE THREAD

**Issue by JesperBrink** [Hogfeldt/bioconda_recipe_gen#47]
When running our sanity check we realised that it didn't work on MacOS. This turns out to be because: "The macOS compilers require the macOS 10.9 SDK". 
This is explained in: https://docs.conda.io/projects/conda-build/en/latest/resources/compiler-tools.html together with a solution.

The questions is: should we handle this, by making a guide to Mac users, or shouldn't we handle it, since it is something that is already documented in conda build?

**Comment by JesperBrink** [comment]
It might be hard for the user to handle this themselves, since our setup method before the sanity check (bioconda_utils_build_setup), creates the folder inside `Bioconda-recipes/recipes` and adds the meta.yaml and build.sh that it have generated, before running the sanity check. 
The user would have to manually add their conda_build_config.yaml into this folder after it is created but before we start running the test (which is right after, making it impossible for the user to do).

**Comment by JesperBrink** [comment]
From this link: https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#conda-build-variant-config-files, it looks like it should be possible for the user to put their conda_build_config.yaml file inside their HOME directory as a permanent location where conda always looks first. This solves the problem introduced in the previous comment.

Furthermore, I just realised that the method I looked at `bioconda_utils_build_setup`isn't the one used for this purpose, but this is instead done in `mini_sanity_check`. (It is actually not used at all. We might want to remove all code that was used before we switched to our own `conda-build-mini`)
